handel/src/services/sqs/params-schema.json

```diff
@@ -0,0 +1,130 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "type": "object",
+    "properties": {
+        "type": {
+            "type": "string",
+            "description": "The service type (should always be 'sqs')",
+            "pattern": "sqs",
+            "errorMessage": "The 'type' parameter must equal 'sqs'"
+        },
+        "query_type": {
+            "type": "string",
+            "description": "The type of queue can be 'regular' and 'fifo'",
+            "pattern": "^regular|fifo$",
+            "errorMessage": "The 'query_type' parameter must equal 'regular' or 'fifo'"
+        },
+        "delay_seconds": {
+            "type": "number",
+            "errorMessage": "The 'delay_seconds' field must be a string"
+        },
+        "content_based_deduplication": {
+            "type": "boolean",
+            "errorMessage": "The 'content_based_deduplication' field must be a boolean"
+        },
+        "max_message_size": {
+            "type": "number",
+            "minimum": 0,
+            "maximum": 262144,
+            "errorMessage": "The 'max_message_size' field must be a number between 0 and 262144"
+        },
+        "message_retention_period": {
+            "type": "number",
+            "minimum": 60,
+            "maximum": 1209600,
+            "errorMessage": "The 'message_retention_period' field must be a number between 60 and 1209600"
+        },
+        "received_message_wait_time": {
+            "type": "number",
+            "minimum": 0,
+            "maximum": 20,
+            "errorMessage": "The 'received_message_wait_time' field must be a nuber between 0 and 20"
+        },
+        "visibility_timeout": {
+            "type": "number",
+            "minimum": 0,
+            "maximum": 43200,
+            "errorMessage": "The 'visibility_timeout' field must be a number between 0 and 43200"
+        },
+        "dead_letter_queue": {
+            "type": "object",
+            "properties": {
+                "max_receive_count":{
+                    "type": "number",
+                    "errorMessage": "The 'max_receive_count' field must be a number"
+                },
+                "delay_seconds": {
+                    "type": "number",
+                    "errorMessage": "The 'delay_seconds' field must be a number"
+                },
+                "max_message_size": {
```

Comment:
I love that you put minimum and maximums on the parameters at the top-level, it would be great if we could add those for the number types in the 'dead_letter_queue' section as well.